CodeOfConduct.md

```diff
@@ -0,0 +1,69 @@
+# Seattle CoffeeOps - Code Of Conduct
+
+## ANTI-HARASSMENT POLICY
+
+CoffeeOps is dedicated to providing a harassment-free
+experience for everyone, regardless of gender, sexual orientation,
+disability, physical appearance, body size, race, or religion. We do not
```

Comment:
You use the oxford comma here, and in one other spot below, but not over the entire document. I'd settle on one, just for consistency.